### DISCUSSION TITLE
如果表单开启了effects，是否就不能使用schema模式的联动协议了？

### DISCUSSION THREAD

**Discussion by wittech** [alibaba/formily#2905] [Q&A 2.x]
目前在启用了表单effects的表单中配置schema模式的联动，报错：Cannot add property effects, object is not extensible
"x-reactions": {
                "target": "disabledTags",
                "fulfill": {
                  "state": {
                    "disabled": "{{$self.value}}"
                  }
                }
              }

**Comment by janryWang** [comment]
具体是啥问题
